src/load_lacounty_parcels.py

```diff
@@ -0,0 +1,85 @@
+"""
```

Comment:
@tiffanychu90 is this now safe to remove?

Reply:
Removed in next commit